docs/source/schema/basic_syntax.rst

```diff
@@ -17,8 +17,7 @@ Portex provides the basic key ``type``. Its value means the type of data and pre
 string. The bulitin supported types can be found in :doc:`/schema/primitive_types` and
 :doc:`/schema/complex_types/index`.
 
-The most important feature of Portex is that the type is configurable, different types has different
-parameters.
+The most important feature of Portex is that types are composable, complex types can be assembled with other types that have already been definited.
```

Comment:
I think `configurable` has a larger scope than `composable`? And here I think the `configurable` is more accurate to describe Portex types. You can find the `configurable` related info in <https://openbytes-schema.readthedocs.io/en/latest/schema/template_type.html>